### PR TITLE
CssUrlRewrite

### PR DESCRIPTION
**Pull request by msbragi**
Hello Mark i tried to follow your guidelines.
Let me know if this is a possible way.
I hope not to be too intrusive. 
If it is not good let me know and i will not insist.
Have a nice day.

### CODE REVIEW

**Comment by Oxicode** [comment]
+1

File: Lib/Filter/CssUrlRewrite.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?php
+App::uses('AssetFilter', 'AssetCompress.Lib');
```
Comment (markstory) [inline Lib/Filter/CssUrlRewrite.php]: Your pull request is targetting the `master` branch, which is a 3.x compatible version of AssetCompress, but this pull request using 2.x conventions. Are you intending to update the plugin for 2.x or for 3.x?

File: Lib/Filter/CssUrlRewrite.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+		 * Add Baseurl to path
+		*/
+		$path = $this->_settings['rewriteUrl'] . str_replace(DS, '/', $path);
+		/*
+		 * Remove trailing slash
+		*/
+		$path = rtrim($path, '/');
+		/*
+		 * Store path for callback
+		*/
```
Comment (markstory) [inline Lib/Filter/CssUrlRewrite.php]: Each line doesn't need a comment.

File: Lib/Filter/CssUrlRewrite.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+			// $matches[1] is either quoted or not
+			$quoteChar = ($matches[1][0] === "'" || $matches[1][0] === '"')	? $matches[1][0] : '';
+			$uri       = ($quoteChar === '') ? $matches[1] : substr($matches[1], 1, strlen($matches[1]) - 2);
+		}
+		// root-relative       protocol (non-data)             data protocol
+		if ($uri[0] !== '/' && $uri[0] !== '#' && strpos($uri, '://') === false && strpos($uri, 'data:') !==  0){
+			$uri = $this->_cleanPath ($this->_path.'/'.$uri);
+		}
+	
+		//var_dump("url({$quoteChar}{$uri}{$quoteChar})");
```
Comment (markstory) [inline Lib/Filter/CssUrlRewrite.php]: This should be deleted.

**Comment by markstory** [comment]
Closing as I thought this was a new pull request, but its a few years old and cannot be merged now.

**Pull request closed without merging**